test/puppetlabs/puppetdb/http/nodes_test.clj

```diff
@@ -433,19 +433,18 @@
           (are-error-response-headers headers)
           (is (re-find msg body)))))))
 
-;TODO fix all skipped tests
-(deftest-http-app ^:skipped tests-for-PDB-5026
+(deftest-http-app query-with-from
   [[version endpoint] endpoints
-  method [:get :post]]
-
-  ; enable this test after ticket https://tickets.puppetlabs.com/browse/PDB-5026 is solved
-  (testing "in a reports subquery"
-    (let [expected ["web1.example.com" "web2.example.com" "puppet.example.com" "db.example.com"]
-          result (query-result method endpoint  ["in", "certname",
-                                                 ["from", "reports",
-                                                  ["extract", "certname"],
-                                                  ["order_by", ["certname"]]]])]
-      (is (= (mapv :certname result) expected)))))
+   method [:get :post]]
+
+  (testing "should change context to reports"
+    (let [expected ["db.example.com" "puppet.example.com" "web1.example.com"]
+          _ (store-example-nodes)
```

Comment:
I think that if you don't use the return value from `store-example-nodes` you can omit  the `_`

Reply:
Sounds reasonable, and alternately, could just replace the `let` entirely, i.e.
```clj
  (testing "should change context to reports"
    (store-example-nodes)
    (is (= ["db.example.com" "puppet.example.com" "web1.example.com"]
           (->> ["in" "certname"
                 ["from" "reports"
                  ["extract" "certname"] ["order_by" ["certname"]]]]
                (query-result method endpoint)
                (mapv :certname result)
                sort)))))
```
But not critical either way.

In any case, I would suggest removing the commas, and flipping the test order from `(is (= actual expected))` to `(is (= expected actual))`, since that's what `clojure.test` expects.